src/interfaces/ILendingPool.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity >=0.4.22 <0.9.0;
 interface ILendingPool {
     function syncInterests() external;
 
-    function borrow(uint256 amount, address vault, address to) external;
+    function borrow(uint256 amount, address vault, address to, bytes3 ref) external;
```

Comment:
Do we actually use this interface at all?

Reply:
no we don't... Let's clean up interfaces & import in another branch